app/representers/api/podcast_representer.rb

```diff
@@ -1,4 +1,67 @@
 # encoding: utf-8
 
 class Api::PodcastRepresenter < Api::BaseRepresenter
+  property :id
```

Comment:
👍!!!  Every couple weeks, I end up trying to use the feeder api to find a podcast.  And it always ends in tears.